tests/data/Alternates-Regular.ufo/fontinfo.plist

```diff
@@ -0,0 +1,291 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+    <key>ascender</key>
+    <integer>1069</integer>
+    <key>capHeight</key>
+    <integer>714</integer>
+    <key>descender</key>
+    <integer>-293</integer>
+    <key>familyName</key>
+    <string>Alternates</string>
+    <key>guidelines</key>
+    <array>
+      <dict>
+        <key>angle</key>
```

Comment:
I wonder if you could trim unneeded pieces of data like the guidelines here, to make this test ufo smaller and more manageable